joint_trajectory_controller/doc/userdoc.rst

```diff
@@ -155,6 +155,11 @@ open_loop_control (boolean)
 
   Default: false
 
+start_holding (bool)
```

Comment:
For more clarity:

```suggestion
start_with_holding (bool)
```

Reply:
Although, I know that I wasn't sure about this – but this might be default behavior without the parameter. I think I confused something before.

Reply:
done